public/assets/js/main.js

```diff
@@ -0,0 +1,56 @@
+$( document ).ready(function(){
+    $('.bxslider').bxSlider({
+      mode: 'fade',
+      captions: true
+    });
+});
+
+function addProduct(prodid) {
```

Comment:
This `addProduct` is added on the window object, this pattern is not suggested, this can pollute the window object. Can you create a separate namespace or implement module pattern to add such functions

Reply:
Created separate namespaces.